core/identity-management/iin-agent/src/common/ledgerBase.ts

```diff
@@ -26,7 +26,7 @@ export class LedgerBase {
     }
 
     // Collect security domain membership info
-    async getSecurityDomainMembership(): Promise<object> {
+    async getSecurityDomainMembership(securityDomain: string): Promise<object> {
```

Comment:
Change argument to `securityDomainId` to indicate it's just the ID and not the structure.